code-submissions-clustering-plugin/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/cli/LoadRunner.kt

```diff
@@ -3,30 +3,47 @@ package org.jetbrains.research.code.submissions.clustering.cli
 import com.intellij.openapi.application.ApplicationStarter
 import com.intellij.openapi.diagnostic.Logger
 import com.xenomachina.argparser.ArgParser
+import com.xenomachina.argparser.default
 import org.jetbrains.kotlinx.dataframe.DataFrame
 import org.jetbrains.kotlinx.dataframe.io.readCSV
+import org.jetbrains.kotlinx.dataframe.io.writeCSV
 import org.jetbrains.research.code.submissions.clustering.load.unifiers.PyUnifier
 import org.jetbrains.research.code.submissions.clustering.util.loadGraph
+import org.jetbrains.research.code.submissions.clustering.util.toDataFrame
+import org.jetbrains.research.code.submissions.clustering.util.toProto
+import java.io.File
 import java.nio.file.Paths
 import kotlin.system.exitProcess
 
 object LoadRunner : ApplicationStarter {
     private val logger = Logger.getInstance(this::class.java)
-    private lateinit var inputFile: String
+    private var outputFilename: String? = null
+    private var csvFilename: String? = null
+    private lateinit var inputFilename: String
 
     override fun getCommandName(): String = "load"
 
     @Suppress("TooGenericExceptionCaught")
     override fun main(args: MutableList<String>) {
         try {
             ArgParser(args.drop(1).toTypedArray()).parseInto(::TransformationsRunnerArgs).run {
-                inputFile = Paths.get(input).toString()
+                inputFilename = Paths.get(input).toString()
+                outputFilename = output?.let { Paths.get(it).toString() }
+                csvFilename = outputCSV?.let { Paths.get(it).toString() }
             }
 
-            val df = DataFrame.readCSV(inputFile)
+            println(csvFilename)
+            val df = DataFrame.readCSV(inputFilename)
             val unifier = PyUnifier()
             val submissionsGraph = df.loadGraph(unifier)
-            println(submissionsGraph.buildStringRepresentation())
+
+            outputFilename?.let {
+                val outputFile = File(it)
+                submissionsGraph.toProto().writeTo(outputFile.outputStream())
+            }
+            csvFilename?.let {
+                submissionsGraph.toDataFrame().writeCSV(it)
+            }
```

Comment:
I think we should add a possibility to choose the final representation and add several cli arguments, e.g. `--serialize`, `--saveCSV`
An example of working with additional arguments into an ij task without new gradle tasks can be found [here](https://github.com/JetBrains-Research/plugin-utilities/blob/main/plugin-utilities-plugin/src/main/kotlin/org/jetbrains/research/pluginUtilities/preprocessing/PreprocessKotlinJava.kt#L11)
What do you think?

Also by default, we also can save into a file a string representation of the graph (submissionsGraph.buildStringRepresentation())